app/models/bliss_attachments/deleter.rb

```diff
@@ -37,6 +37,7 @@ def respond_to?(meth, include_private=false)
     def destroy_attachment(attachments, file_path)
       attachment = attachments
         .find { |a| a.file.url(:large, false) == without_cache_key(file_path) }
+      return if attachment.nil?
```

Comment:
Please add a newline after the guard clause.

Reply:
done